docs/source/autoscaling.rst

```diff
@@ -53,12 +54,14 @@ Decision policies
 The currently available decicion policies are:
 
 :pid:
-  Uses a PID controller to determine when to autoscale a service.
+  Uses a PID controller to determine when to autoscale a service. See https://en.wikipedia.org/wiki/PID_controller for more information on PIDs.
 
   Autoscaling parameters:
 
   :setpoint: the target utilization the controller aims for. Defaults to 0.8 (80%).
-  :delay: the number of seconds the decision policy must wait before fetching new data. Defaults to 600.
+  :Kp: the proportional control constant. Defaults to 0.2.
+  :Ki: the integral control constant. Defaults to 0.2.
```

Comment:
Tend to agree with @solarkennedy . These controls are not something most services will need. If we find a workflow that requires such fine grain control, we probably want to talk to them to better understand their usecase rather than just having them adjust these settings.